receiver/stanzareceiver/receiver.go

```diff
@@ -25,6 +25,7 @@ import (
 
 type stanzareceiver struct {
 	agent    *stanza.LogAgent
+	emitter  *LogEmitter
```

Comment:
This doesn't seem to be used anywhere.

Reply:
That's true. Wasn't strictly necessary in this PR. It'll be used in the next one, which will be ready shortly.